crates/dojo-core/src/interfaces.cairo

```diff
@@ -19,6 +28,18 @@ trait IWorld {
     fn delete_entity(component: ShortString, query: Query);
 }
 
+// Remove once Serde is derivable for dispatchers
+impl IWorldDispatcherSerde of Serde<IWorldDispatcher> {
+    fn serialize(self: @IWorldDispatcher, ref output: Array<felt252>) {
+        output.append((*self.contract_address).into());
+    }
+    fn deserialize(ref serialized: Span<felt252>) -> Option<IWorldDispatcher> {
+        let contract_address: felt252 = *serialized.pop_front()?;
+        let contract_address: ContractAddress = contract_address.try_into().unwrap();
+        Option::Some(IWorldDispatcher { contract_address })
+    }
+}
+
```

Comment:
TODO: remove this once https://github.com/starkware-libs/cairo/pull/3241 is merged